src/components/toolTip/ToolTip.module.css

```diff
@@ -0,0 +1,49 @@
+:root {
+  --conduction-size-xs: var(--skeleton-size-xs);
+  --conduction-color-white: var(--skeleton-color-white);
+  --conduction-color-black: var(--skeleton-color-black);
+  --conduction-font-size-sm: var(--skeleton-font-size-sm);
+  --conduction-content-font-family: var(--skeleton-content-font-family);
+  --conduction-line-height-xs: var(--skeleton-line-height-xs);
+  --conduction-font-size-4xs: var(--skeleton-font-size-4xs);
+  --conduction-size-xl: var(--skeleton-size-xl);
```

Comment:
Using design tokens to set all these values is great. However, we're creating too generic token keys.

Right now, when implementing this Tooltip within any implementation (e.g. OpenCatalogi) we have to override `--conduction-size-xs` to update the Tooltip. There's no way of knowing that this is regarding the Tooltip. Another example would be: `--conduction-content-font-family`, _what_ content is this? No way to know within the implementation layer.

This would result into changing these tokens to (at least) a prefixed version, so:

Not this: `--conduction-content-font-family`;
But this: `--conduction-tooltip-content-font-family`;

After prefixing, it's also best to very explicitly set _what_ the token will actually be doing, like so:

```
--conduction-size-xs; // current without prefix
--conduction-tooltip-size-xs; // with prefix

--conduction-tooltip-padding-inline-start; // with prefix and declarative;
--conduction-tooltip-padding-inline-end; // with prefix and declarative;
--conduction-tooltip-padding-block-start; // with prefix and declarative;
--conduction-tooltip-padding-block-start; // with prefix and declarative;
```

The same goes for every token. Everything should be very descriptive.